src/main/scala/org/broadinstitute/dsde/workbench/sam/api/TermsOfServiceRoutes.scala

```diff
@@ -120,7 +121,10 @@ trait TermsOfServiceRoutes extends SamUserDirectives with SamRequestContextDirec
                 pathEndOrSingleSlash {
                   getWithTelemetry(samRequestContext, userIdParam(requestUserId)) {
                     complete {
-                      tosService.getTermsOfServiceDetailsForUser(requestUserId, samRequestContext)
+                      tosService.getTermsOfServiceDetailsForUser(requestUserId, samRequestContext).map {
+                        case response @ TermsOfServiceDetails(None, None, false, false) => StatusCodes.NotFound -> response
+                        case response => StatusCodes.OK -> response
```

Comment:
Sorry for being a curmudgeon. Can `getTermsOfServiceDetailsForUser` itself return an `Optional`? The "Combined" response should modify the `Optional.Empty` to be what it expects.

Reply:
ok yeah that sounds good no worries